starter-code/src/functions-and-arrays.js

```diff
@@ -40,6 +83,22 @@ const wordsUnique = [
   'bring'
 ];
 
+ function uniquifyArray(arr){
```

Comment:
Esta les quedo incompleta, pero andaban por buen camino,
Les falto agregar un condicional para checar si el numero de caracteres es mayor que 0, que pase algo.
En su **for()** designaron el index como 1, en este caso se debio empezar en el numero 0 o se va a brincar una palabra.


`function uniquifyArray(arr) {`
 ` let newArray = []`
 ` if (arr.length > 0) {`
  `  for (let i = 0; i < arr.length; i++) {`
   `   if (newArray.includes(arr[i]) == false) {`
       ` newArray.push(arr[i])`
      `} else {`
       ` continue`
    `  }`
`    }`
 `   return newArray`
 ` } else {`
  `  return []`
 ` }`
`}`